uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/NetworkAuthenticationChallengeRelay.cpp

```diff
@@ -0,0 +1,62 @@
+
+/*******************************************************************************
+ *  Copyright 2012-2024 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "NetworkAuthenticationChallengeRelay.h"
+#include "ArcGISAuthenticationController.h"
+
+// Maps SDK headers
+#include <Authentication/NetworkAuthenticationChallenge.h>
+#include <Authentication/AuthenticationManager.h>
+#include <ArcGISRuntimeEnvironment.h>
+
+using namespace Esri::ArcGISRuntime;
+using namespace Esri::ArcGISRuntime::Authentication;
+
+namespace Esri::ArcGISRuntime::Toolkit {
+
+/*!
+ \class Toolkit::NetworkAuthenticationChallengeRelay
+ \inmodule EsriArcGISRuntimeToolkit
+ \ingroup ArcGISQtToolkitUiCppControllers
+ \brief Simple implementation of NetworkAuthenticationChallengeHandler.
```

Comment:
same comment here about the brief. Also, Handler -> Relay with the rename.

Reply:
It is an implementation of NetworkAuthenticationChallengeHandler, the interface from the API. I will reword it to make that clearer.